addons/artillerytables/functions/fnc_rangeTableOpen.sqf

```diff
@@ -65,7 +65,7 @@ GVAR(magModeData) = [];
 {
     _x params ["_xDisplayNameShort", "_xDisplayName", "_xInitSpeed", "_xAirFriction"];
     if (_allSameCharge) then {
-        _ctrlChargeList lbAdd format ["%1", _xDisplayNameShort];
+        _ctrlChargeList lbAdd str _xDisplayNameShort;
```

Comment:
```
format ["%1", "test"]; = "test"
str "test"; = """test"""
```

in this case we can easily do the following, but I'm not sure if that's always true
```suggestion
        _ctrlChargeList lbAdd _xDisplayNameShort;
```

Reply:
Ah, interesting